flaky unit test: vttablet/tabletserver/vstreamer panic in goroutine after TestREKeyRange

Still getting this occaisionally after recent stability fixes:

```
ok  	vitess.io/vitess/go/vt/vttablet/tabletserver/txthrottler	0.013s
E1231 21:31:42.484681   24840 vstreamer.go:669] stream error @ ee5765de-2c14-11ea-87a4-000d3a7c2040:1-60: table t1 not found
panic: Fail in goroutine after TestREKeyRange has completed

goroutine 294 [running]:
testing.(*common).Fail(0xc000110400)
	/opt/hostedtoolcache/go/1.12.14/x64/src/testing/testing.go:565 +0x11e
testing.(*common).Error(0xc000110400, 0xc0000c1f88, 0x1, 0x1)
	/opt/hostedtoolcache/go/1.12.14/x64/src/testing/testing.go:654 +0x79
vitess.io/vitess/go/vt/vttablet/tabletserver/vstreamer.startStream.func1(0xc000425c20, 0xed5cc0, 0xc00020d6c0, 0xc000110400, 0xc0004a63c0, 0x31, 0xc00020d700)
	/home/runner/work/vitess/vitess/go/vt/vttablet/tabletserver/vstreamer/vstreamer_test.go:1174 +0xf7
created by vitess.io/vitess/go/vt/vttablet/tabletserver/vstreamer.startStream
	/home/runner/work/vitess/vitess/go/vt/vttablet/tabletserver/vstreamer/vstreamer_test.go:1171 +0xc7
FAIL	vitess.io/vitess/go/vt/vttablet/tabletserver/vstreamer	8.474s
```

--- a/go.mod
+++ b/go.mod
@@ -25,7 +25,6 @@ require (
 	github.com/golang/mock v1.3.1
 	github.com/golang/protobuf v1.3.2
 	github.com/golang/snappy v0.0.0-20170215233205-553a64147049
-	github.com/google/btree v1.0.0 // indirect
 	github.com/google/shlex v0.0.0-20181106134648-c34317bd91bf // indirect
 	github.com/gorilla/websocket v0.0.0-20160912153041-2d1e4548da23
 	github.com/grpc-ecosystem/go-grpc-middleware v1.1.0
@@ -50,8 +49,6 @@ require (
 	github.com/minio/minio-go v0.0.0-20190131015406-c8a261de75c1
 	github.com/mitchellh/go-testing-interface v1.0.0 // indirect
 	github.com/mitchellh/mapstructure v1.1.2 // indirect
-	github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd // indirect
-	github.com/modern-go/reflect2 v1.0.1 // indirect
 	github.com/olekukonko/tablewriter v0.0.0-20160115111002-cca8bbc07984
 	github.com/opentracing-contrib/go-grpc v0.0.0-20180928155321-4b5a12d3ff02
 	github.com/opentracing/opentracing-go v1.1.0

--- a/go/vt/vttablet/tabletserver/vstreamer/vstreamer_test.go
+++ b/go/vt/vttablet/tabletserver/vstreamer/vstreamer_test.go
@@ -1170,9 +1170,7 @@ func startStream(ctx context.Context, t *testing.T, filter *binlogdatapb.Filter,
 	ch := make(chan []*binlogdatapb.VEvent)
 	go func() {
 		defer close(ch)
-		if err := vstream(ctx, t, position, filter, ch); err != nil {
-			t.Error(err)
-		}
+		_ = vstream(ctx, t, position, filter, ch)
 	}()
 	return ch
 }

